std_msgs/Header header
string frame_id
int32 width
int32 height
int32 depth
ugv_nav4d_ros2/TravPatch[] patches